README.md

```diff
@@ -16,19 +16,20 @@ up. Where possible checks should be automated and have shareable configs.
 
 ## ESLint Configuration
 
-Our ESLint configuration is derived from
-[Airbnb's](https://github.com/airbnb/javascript).
+Our ESLint configuration follows that of
+[Airbnb](https://github.com/airbnb/javascript).
 
 It comes in different flavours:
 
-1. [`eslint-config-bitcrowd-base`](packages/eslint-config-bitcrowd-base)
-  - provides our base linting configuration for ECMAScript 6+ applications
-  - contains alternate config for applications written in ES5 and below
-2. [`eslint-config-bitcrowd-react`](packages/eslint-config-bitcrowd-react)
-  - for ES6+ linting and additional rules for React applications
-3. [`eslint-config-bitcrowd-angular`](packages/eslint-config-bitcrowd-angular)
-  - for ES6+ linting and additional rules for Angular applications
-  - for ES5 and below linting and additional rules for Angular applications
+1. a [base configuration](templates/base/.eslintrc.json) for ECMAScript 6+ code
+2. a [legacy configuration](templates/legacy/.eslintrc.json) for code written
+   in ES5 and below
+2. a [react configuration](packages/react/.eslintrc.json) for ES6+ linting with
+   additional rules for React applications
+3. an [angular configuration](packages/angular/.eslintrc.json)
+
+    for ES6+ linting and additional rules for Angular applications
+    for ES5 and below linting and additional rules for Angular applications
 
 Install the package which fits your current project and create an `.eslintrc`
 file ([example](templates/.eslintrc.json)). We recommend setting `"root": true`
```

Comment:
could it be useful explain why we recommend to set `"root": true` or linking the[ documentation ](http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) ?